mindsdb/integrations/handlers/anyscale_endpoints_handler/anyscale_endpoints_handler.py

```diff
@@ -82,8 +82,9 @@ def predict(self, df: pd.DataFrame, args: Optional[Dict] = None) -> pd.DataFrame
 
     def finetune(self, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
         with self._anyscale_base_api(args):
-            self._set_models(args.get('using', {}))
-            super().finetune(df, args)
+            using_args = args.get('using', {})
+            self._set_models(using_args)
+            super().finetune(df, using_args)
```

Comment:
Curious as to why this was done? It introduced a bug where `target` may not be present. Shouldn't we still send `args` proper to `finetune()` instead?